### ISSUE TITLE
Division by zero

### ISSUE THREAD

**Issue by DimaDDM** [bertrandmartel/speed-test-lib#30]
When i'm trying upload file to ftp server(using startUpload, not startUploadFTP) app crash with

java.lang.ArithmeticException: Division by zero
                                                     at java.math.BigDecimal.divide(BigDecimal.java:1065)
                                                     at fr.bmartel.speedtest.SpeedTestTask.getReport(SpeedTestTask.java:774)
                                                     at fr.bmartel.speedtest.SpeedTestSocket.getLiveDownloadReport(SpeedTestSocket.java:678)
                                                     at fr.bmartel.speedtest.SpeedTestTask.startSocketDownloadTask(SpeedTestTask.java:484)
                                                     at fr.bmartel.speedtest.SpeedTestTask.access$1300(SpeedTestTask.java:57)
                                                     at fr.bmartel.speedtest.SpeedTestTask$2.run(SpeedTestTask.java:409)
                                                     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
                                                     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
                                                     at java.lang.Thread.run(Thread.java:856)

**Comment by bertrandmartel** [comment]
What kind of FTP server are you using ? I tried with : 

```
final String fileName = SpeedTestUtils.generateFileName() + ".txt";
speedTestSocket.startUpload("speedtest.tele2.net", "/upload/" + fileName, 1000000);
```

which will end up with `404` status code as expected but no crash

**Comment by bertrandmartel** [comment]
This has been fixed in 1.30 thank you for you feedback :)
